packages/apps/shopify-app-remix/src/server/authenticate/admin/scope/query.ts

```diff
@@ -17,38 +17,33 @@ export function queryScopesFactory(
   return async function queryScopes() {
     const {logger} = params;
 
-    logger.debug('Query scopes information: ', {
+    logger.debug('Querying scopes details: ', {
       shop: session.shop,
     });
 
-    const scopesInformation = await fetchScopeInformation(admin);
-    return mapFetchScopeInformation(scopesInformation);
+    const scopesDetail = await fetchScopeDetail(admin);
+    return mapFetchScopeDetail(scopesDetail);
   };
 }
 
-export function mapFetchScopeInformation(
-  fetchScopeInformation: FetchScopeInformationResponse,
-): ScopesInformation {
-  const appInformation = fetchScopeInformation.app;
-  const declaredRequired = new AuthScopes(
+export function mapFetchScopeDetail(
+  scopesDetailResponse: FetchScopesDetailResponse,
+): ScopesDetail {
+  const appInformation = scopesDetailResponse.app;
+
+  const granted = new AuthScopes(
+    appInformation.installation.accessScopes.map((scope) => scope.handle),
+  );
+  const required = new AuthScopes(
     appInformation.requestedAccessScopes.map((scope) => scope.handle),
   );
-
-  const grantedRequired = appInformation.installation.accessScopes
-    .map((scope) => scope.handle)
-    .filter((scope) => declaredRequired.has(scope));
-
-  const grantedOptional = appInformation.installation.accessScopes
-    .map((scope) => scope.handle)
-    .filter((scope) => !declaredRequired.has(scope));
+  const optional = new AuthScopes(
+    appInformation.optionalAccessScopes.map((scope) => scope.handle),
+  );
 
   return {
-    granted: {
-      required: grantedRequired,
-      optional: grantedOptional,
-    },
-    declared: {
-      required: declaredRequired.toArray(),
-    },
+    granted: granted.toArray(),
+    required: required.toArray(),
+    optional: optional.toArray(),
```

Comment:
I updated the return value of this method to match the ["ScopesDetail" return value](https://github.com/Shopify/ui-api-design/blob/main/apis/Scopes/README.md#check-read_customers-scope)